cpp/includes/defines.h

```diff
@@ -644,6 +642,8 @@ boundHandling : str
     [DREAM] How steps past the space boundaries should be handled. Can be 'off', 'reflect', 'bound', or 'fold'.
 adaptPCR : bool
     [DREAM] Whether the crossover probability for differential evolution should be adapted during the run.
+calcSLD : bool
+    Whether SLD will be calculated (for live plotting etc.)
```

Comment:
please remove `calcSLD` from Controls, I think you can safely revert all changes in this file.